schemas/resources/schemas/visualization-draft/v1.1.0/index.json

```diff
@@ -0,0 +1,28 @@
+{
+  "MetadataSpecification": {
+    "URL": "https://cdn.earthdata.nasa.gov/generic/index/v0.0.1",
+    "Name": "Generic-Index",
+    "Version": "0.0.1"
+  },
+  "Generic": {
+    "Name": "Visualization Draft",
+    "Version": "1.1.0"
+  },
+  "SubConceptType": "VID",
```

Comment:
Should this be `VISD`? Other draft types add `D` to the published concept's prefix.

Reply:
We have only gone up to 3 letters deep; data quality summary draft for example is DQD, collection draft is CD, tools is TD, variable os VD, services is VD, grid is GD, order option draft is OOD.  I can change it, but then it doesn't follow what data quality summary draft is doing. It is more consistent now.